src/app/pages/common/pages-not-found-page/pages-not-found-page.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container not-found">
+    <div class="app-logo-div">
+        <img class="app-logo" src="assets/images/logos/404-not-found-logo.svg" alt="404-Not-Found-Page-Logo"/>
```

Comment:
Alt text for images should have meaning. It should not be dashed phrase. Here for example it can be "Image for 404 page"